files/find.py

```diff
@@ -313,7 +313,7 @@ def main():
                        continue
 
                     try:
-                        st = os.stat(fsname)
+                        st = os.lstat(fsname)
```

Comment:
i'm thinking we emulate the find command, follow does both directory paths and files (make note of change in description).

another thing i was thinking was about expanding types (symlink, devices, fifos, etc) right now 'file' means 'regular file'